SprawdzPogode/Handlers/ExcelHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Office.Interop.Excel;
+
+namespace SprawdzPogode.Handlers
+{
+    public class ExcelHandler : IOutputHandler
+    {
+        public Application Excel { get; set; }
+        public string Path { get; set; }
+
+        public ExcelHandler(string path, Application excel)
+        {
+            Excel = excel;
+            Path = path;
+        }
+
+        public void Start()
+        {
+            Excel.Visible = true;

```

Comment:
I think it's proper place to check if Excel is already openeded, if yes - then we should close it so the program will work.